tests/addons/restmail.js

```diff
@@ -21,7 +21,7 @@ define([
 
     return self.request.send('/mail/' + user, 'GET')
       .then(function (result) {
-        if (result.length === number) {
+        if (result.length >= number) {
```

Comment:
Yeah, there was some strangeness with the mock emails that I didn't know about - if you have too many mock emails result.length > number and the tests timeout.